ui-support-transfer-obj/src/main/java/org/dcsa/uisupport/transferobjects/TerminalTO.java

```diff
@@ -0,0 +1,20 @@
+package org.dcsa.uisupport.transferobjects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+import javax.validation.constraints.Size;
+
+public record TerminalTO(
+  @Size(max = 100) @JsonProperty("facilityName")
+  String name,
+  @Size(max = 4) @JsonProperty("facilityBICCOde")
```

Comment:
`facilityBICCOde` -> `facilityBICCode`

Reply:
(It can also be removed - the UI does not use BIC codes at all.

Reply:
fixed